docs/site/reference/reserved-binding-keys.md

```diff
@@ -7,7 +7,7 @@ sidebar: lb4_sidebar
 permalink: /doc/en/lb4/Reserved-binding-keys.html
 ---
 
-When using [dependency injection](Dependency-injection.md) there are a few
+When using [dependency injection](Dependency-injection.html) there are a few
```

Comment:
we're trying to consistently use the .md file of the docs. Could you please update it accordingly? Thanks. 
Here is the corresponding md file: https://github.com/strongloop/loopback-next/blob/master/docs/site/Dependency-injection.md.

Reply:
As what @dhmlau as mentioned, let's reference the markdown file directly.

```suggestion
When using [dependency injection](../Dependency-injection.md) there are a few
```